src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,154 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+ class Duke {
+    static Scanner sc = new Scanner(System.in);
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    public static String logo = "    ____        _        \n"
+            + "    |  _ \\ _   _| | _____ \n"
+            + "    | | | | | | | |/ / _ \\\n"
+            + "    | |_| | |_| |   <  __/\n"
+            + "    |____/ \\__,_|_|\\_\\___|\n";
+    private static String[] commands = {"todo", "event", "deadline"};
+
+    public static void echo(String command) {
+        System.out.println("    ____________________________________________________________");
```

Comment:
Do consider making a 'decorator' function that can neatly print any output you want to give to the user, with appropriate formatting, divider bars, and newlines.